drivers/at86rf215/include/at86rf215_internal.h

```diff
@@ -46,8 +46,8 @@ extern "C" {
 /**
  * This is used to calculate the csma backoff based on the bitrate.
  */
-/** 20 symbols is the std period length */
-#define AT86RF215_BACKOFF_PERIOD_IN_SYMBOLS (20U)
+/** 8 symbols is the std period length */
+#define AT86RF215_BACKOFF_PERIOD_IN_SYMBOLS (8U)
```

Comment:
Why this change?

Reply:
Meh, both are wrong :weary: 

![image](https://user-images.githubusercontent.com/1301112/96701462-eac61300-1390-11eb-9b95-10afae9bc5a2.png)

with 

#### aTurnaroundTime

![image](https://user-images.githubusercontent.com/1301112/96701774-41cbe800-1391-11eb-8321-98e0bc72c5bc.png)

#### aCCATime

> For the SUN PHYs other than SUN O-QPSK, the duration of 8 symbol periods, as defined in 6.1. For the SUN O-QPSK PHY, this value is defined in Table 21-24. For all other PHYs, the duration of 8 symbol periods.

![image](https://user-images.githubusercontent.com/1301112/96701909-6cb63c00-1391-11eb-8ecb-ca6c4614f1df.png)

Reply:
should be fixed now.